akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/R2dbcProjectionImpl.scala

```diff
@@ -362,6 +363,17 @@ private[projection] object R2dbcProjectionImpl {
                 Some(loadedEnvelope)
               }
             } else {
```

Comment:
There is also the case when isAccepted returns a failed Future. That is when a backtracking event is not accepted. For that case it's even more important to trigger a replay, and not fail the stream.

Reply:
Did something in 97140ae64b417d0e039c21a09c54da03304af1b2 but not sure if it is the right thing.